x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/properties/table.tsx

```diff
@@ -128,39 +128,46 @@ const ArrayValue = (props: MoreProps) => {
     setIsExpanded(false);
   }, []);
 
+  const list = Array.isArray(values) ? values : [values];
+  const [first, ...others] = list;
+  const hasOthers = others.length > 0;
+  const shouldShowMore = hasOthers && !isExpanded;
+
   return (
-    <>
-      {!isExpanded && (
-        <EuiFlexGroup gutterSize={'xs'} responsive={false} alignItems={'baseline'} wrap={true}>
-          <EuiFlexItem grow={false}>
-            {first(values)}
+    <EuiFlexGroup
+      gutterSize={'xs'}
+      responsive={false}
+      alignItems={'baseline'}
+      wrap={true}
+      direction="column"
+    >
+      <div>
+        {first}
+        {shouldShowMore && (
+          <>
             {' ... '}
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiLink data-test-subj="infraArrayValueCountMoreLink" onClick={expand}>
+            <EuiLink data-test-subj="infraExpandableContentCountMoreLink" onClick={expand}>
```

Comment:
Is it safe to change the `data-test-subj`? Could there possibly be something expecting it to be id on the analytics side?

Reply:
same here